clusterloader2/pkg/measurement/common/network/controller.go

```diff
@@ -0,0 +1,439 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package network
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"math"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	//"github.com/montanaflynn/stats"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
+)
+
+type controller struct {
```

Comment:
Actually - I think that solving https://github.com/kubernetes/perf-tests/issues/1631 would mean that we would no longer need this controller, as we will be able to do everything from the measurement.

So I actually think that before we will make this PR, we should resolve the first - it will simplify this PR a lot!